components/webapps/blacklist-whitelist-service/src/main/java/com/wso2telco/services/bw/Queries.java

```diff
@@ -418,12 +315,13 @@ public Response getAllSubscribers() throws SQLException, BusinessException {
 
 	@GET
 	@Path("/apps/{userId}/{operatorId}")
-	@Produces("application/json")
+	@Produces(MediaType.APPLICATION_JSON)
 	public Response getAllApplicationsByUser(@PathParam("userId") String userId,@PathParam("operatorId") String operatorId) throws SQLException, BusinessException {
```

Comment:
remove the SQLException which is not thrown from the method

Reply:
updated